km61/Morozov_Artem/laboratory1.sql

```diff
@@ -1,2 +1,55 @@
--- LABORATORY WORK 1
--- BY Morozov_Artem
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+створювати таблиці
+4 бали
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE user student
+IDENTIFIED by Artem;
+GRANT create table to  Artem;
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Студент має мобільний номер іноземного оператора.
+4 бали
+---------------------------------------------------------------------------*/
+--Код відповідь:
+CREATE TABLE  student_name (
+  s_name VARCHAR (30), not null;
+  telephon_number integer (10), not null;
+);
+/* --------------------------------------------------------------------------- 
+3. Надати додаткові права користувачеві (створеному у пункті № 1) для створення таблиць, 
+внесення даних у таблиці та виконання вибірок використовуючи команду ALTER/GRANT. 
+Згенерувати базу даних використовуючи код з теки OracleScript та виконати запити: 
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+alter user Artem
+grant insert any to Artem;
+grant select any to Artem;
```

Comment:
It's incorrect...
1).: incorrect syntax
2). In user which you've created- Artem it's a password. OMG(((9.
Correct answer:
grant create any table to student
grant insert any table to student;
grant select any table to student;
Yeaah, not for Artem, because your user  was named "student"(((9.